containers/gds/Dockerfile

```diff
@@ -1,22 +1,51 @@
-FROM golang:1.16 AS builder
+# Dynamic Builds
+ARG BUILDER_IMAGE=golang:1.16-buster
+ARG FINAL_IMAGE=debian:buster-slim
 
-WORKDIR /srv/build
+# Build stage
+FROM ${BUILDER_IMAGE} as builder
 
-COPY . .
-RUN go test ./... && go build -v ./cmd/gds
-RUN go build -v ./cmd/sectigo
-RUN go build -v ./cmd/gdsutil
+# Ensure ca-certificates are up to date
+RUN update-ca-certificates
```

Comment:
Was this not something we were doing before? Could you clarify how this differs from the steps on lines 41-44?

Reply:
We were not doing this before; though it's unlikely that it would cause a problem, it could future-proof dependency downloads in the future. I figured it wouldn't hurt to help us maintain these Dockerfiles in the long run, particularly if we move to distroless or static (the blog post that I posted in the #engineering channel a month or two ago). 

This command is being run in the build stage, and ensures that the certs are up to date for any downloads happening in this stage such as dependency downloads.  Lines 41-44 are in the final stage (which is a different container) - those lines are more about ensuring that TLS certs (such as mTLS certs) can be correctly verified.